<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gotická Dáma</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/icons/window.ico</normaloff>:/images/icons/window.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>541</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>541</width>
      <height>541</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>541</width>
      <height>541</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="dragMode">
     <enum>QGraphicsView::NoDrag</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>331</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>Historie tahů</string>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>137</red>
        <green>91</green>
        <blue>46</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGotickaDama">
    <property name="title">
     <string>Soubor</string>
    </property>
    <addaction name="actionNovaHra"/>
    <addaction name="actionOtevritHru"/>
    <addaction name="separator"/>
    <addaction name="actionUlozit"/>
    <addaction name="actionUlozitHruJako"/>
    <addaction name="separator"/>
    <addaction name="actionUkoncit"/>
   </widget>
   <widget class="QMenu" name="menuPravo">
    <property name="title">
     <string>Úpravy</string>
    </property>
    <addaction name="actionVratitTah"/>
    <addaction name="actionZnovu"/>
    <addaction name="separator"/>
    <addaction name="actionNastaven"/>
   </widget>
   <widget class="QMenu" name="menuNapoveda">
    <property name="title">
     <string>Nápověda</string>
    </property>
    <addaction name="actionZobrazitNapovedu"/>
    <addaction name="actionNapovedaNejlepsihoTahu"/>
    <addaction name="separator"/>
    <addaction name="actionOProgramu"/>
   </widget>
   <addaction name="menuGotickaDama"/>
   <addaction name="menuPravo"/>
   <addaction name="menuNapoveda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
  <action name="actionNovaHra">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/icons/add.png</normaloff>:/images/icons/add.png</iconset>
   </property>
   <property name="text">
    <string>Nová hra</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOtevritHru">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/icons/open.png</normaloff>:/images/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Otevřít hru...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionUlozit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/icons/save.png</normaloff>:/images/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Uložit hru</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionUlozitHruJako">
   <property name="text">
    <string>Uložit hru jako...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionVratitTah">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/icons/undo.png</normaloff>:/images/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Zpět</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionNastaven">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/icons/gear.png</normaloff>:/images/icons/gear.png</iconset>
   </property>
   <property name="text">
    <string>Nastavení hry</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionZobrazitNapovedu">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/icons/question.png</normaloff>:/images/icons/question.png</iconset>
   </property>
   <property name="text">
    <string>Zobrazit nápovědu</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionOProgramu">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/icons/info.png</normaloff>:/images/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>O programu</string>
   </property>
  </action>
  <action name="actionUkoncit">
   <property name="text">
    <string>Ukončit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNapovedaNejlepsihoTahu">
   <property name="text">
    <string>Nápověda nejlepšího tahu</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionZnovu">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/icons/redo.png</normaloff>:/images/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Znovu</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
